### DISCUSSION TITLE
MC崩溃解决方案

### DISCUSSION THREAD

**Discussion by su2569** [share121/inter-knot#1670] [其他]
![HMCL](https://f.cdn-static.cn/81470_17234313395253.png)
> [!TIP]
> 这只是小部分的解决方案

HMCL游戏崩溃后在崩溃界面点左下角「导出游戏崩溃信息」（不要点「日志」）
PCL游戏崩溃后在崩溃界面点左下角「导出错误报告」（不要点「日志」）
如果启动器有提示，最好先看提示（看不懂就去翻译）

如果有这么弹出一个窗口的话
![GLFWerror65543](https://i.mcmod.cn/editor/upload/20240503/1714745073_171385_ckMc.webp)
请看修复GLFW error 65543{
https://www.mcmod.cn/class/15441.html `(Forge:1.21~1.18、1.17.1 Fabric:1.21~1.20)`
https://www.mcmod.cn/class/6765.html `(Forge:1.20.4~1.18、1.17.1 Fabric:1.20.4~1.18、1.17.1、1.17 Quilt:1.19.4~1.18、1.17.1、1.17)`
}，如果是这个
![GLFWerror65542](https://tiebapic.baidu.com/forum/w%3D580/sign=2282392fb30f4bfb8cd09e5c334e788f/55ae0bb30f2442a779b3ab15c643ad4bd31302c4.jpg?tbpicau=2024-08-23-05_c29c6eb9db63270fb5a3914a394552c8)
请看https://tieba.baidu.com/p/7333764519
如果里面没有或不想看，这里有方案（记得看显卡型号 https://jingyan.baidu.com/article/9080802219bcfcbc91c80fe0.html ）
> 笔记本电脑使用独立显卡启动 Minecraft 教程：https://blog.beanbang.cn/2017/07/10/gui-config-for-minecraft/
> 在 NVIDIA 控制面板里按图里顺序改成「高性能 NVIDIA 处理器」，然后点下面的「应用」，最后重启电脑 https://gitcode.net/chearlai/kookfiles/-/raw/master/assets/2023-06/LHqkHeKbD90pr0aw.png
> AMD 独显开启教程：https://zhuanlan.zhihu.com/p/548563759
> 下载安装这个 AMD 核显驱动： https://gitcode.net/chearlai/kookfiles/-/raw/master/assets/2024-01/svhIMpWsDS14c0iu.png
> 下载安装这个 AMD 显卡驱动：https://www.amd.com/en/support/kb/release-notes/rn-rad-win-22-5-1 https://gitcode.net/chearlai/kookfiles/-/raw/master/assets/2023-06/VoQjjMQlTg0x905x.png
> Intel 显卡驱动列表：https://www.intel.com/content/www/us/en/support/articles/000090440/graphics.html
> AMD 显卡驱动列表：https://www.amd.com/en/support
> NVIDIA 显卡驱动列表：https://www.nvidia.com/download/index.aspx
> 按照图里的顺序把「使用 OpenGL 软渲染器」打开 https://gitcode.net/chearlai/kookfiles/-/raw/master/assets/2023-04/SuAn2sjIow0md0b6.png https://gitcode.net/chearlai/kookfiles/-/raw/master/assets/2023-06/06kkZoEmeP0m2061.png https://gitcode.net/chearlai/kookfiles/-/raw/master/assets/2023-01/dODEb69kiV0gg09h.png
> 在启动器的游戏设置里的高级设置里把渲染器选成软渲染器 https://gitcode.net/chearlai/kookfiles/-/raw/master/assets/2023-04/SuAn2sjIow0md0b6.png https://gitcode.net/chearlai/kookfiles/-/raw/master/assets/2023-06/06kkZoEmeP0m2061.png https://gitcode.net/chearlai/kookfiles/-/raw/master/assets/2023-07/DAVItvJ3Z90mc093.png https://gitcode.net/chearlai/kookfiles/-/raw/master/assets/2023-03/j9n8YaA5Ik0m20dk.png
> OpenGL版本太低解决：https://www.bilibili.com/video/BV1Ch4y1k7Ki [doge]
如果没有，自己先解包看一下（以下用HMCL作为例子）
先看有没有hs_err开头的文件
如果有就点开
> 看第四行是否为EXCEPTION_ACCESS_VIOLATION (0xc0000005)
> 如果是就看看是不是甲骨文（微软）java/区域是不是中国/路径有没有问题{
> 甲骨文（微软）java：之前的卸载，然后

> [!WARNING]
> 按照需求下载！( https://www.bilibili.com/read/cv35088159/ )
> 下载安装这个 Java，安装的时候不要改安装位置：https://download.bell-sw.com/java/21.0.4+9/bellsoft-jre21.0.4+9-windows-aarch64-full.msi
> 下载安装这个 Java，安装的时候不要改安装位置：https://download.bell-sw.com/java/8u422+6/bellsoft-jre8u422+6-windows-amd64-full.msi
> 下载安装这个 Java，安装的时候不要改安装位置，https://download.bell-sw.com/java/11.0.13+8/bellsoft-jre11.0.13+8-windows-amd64-full.msi 
> 下载安装这个 Java，安装的时候不要改安装位置，https://download.bell-sw.com/java/17.0.12+10/bellsoft-jre17.0.12+10-windows-amd64-full.msi
> 区域：在 控制面板 -- 时钟和区域 -- 区域 -- 管理 -- 更改系统区域设置 里面上面选「中文（简体，中国）」，并且把下面那个勾去掉 https://gitcode.net/chearlai/kookfiles/-/raw/master/assets/2023-03/zsLRF3LrcD0we0hj.png
> 路径：改成无符号（除了.minecraft）无中文的路径
> 其他的就不会了（悲

然后看hmcl.log，第一行请确保最后面是`*** HMCL 3.5.9 ***`
如果不是就手动去 https://zkitefly.github.io/HMCL-Snapshot-Update/prs 下载新版本，因为不是的话可能会报错“错误：找不到或无法加载主类 net.minecraft.client.main.Main 原因: java.lang.ClassNotFoundException:net.minecraft.client.main.Main”（该原因其他方案 https://tieba.baidu.com/p/7693682857 ）
~~之后看launch.bat（算了，基本都看不懂，不发了）~~
最后看一遍压缩包（如果有一大片的crash，那你是在干什么啊喂！）和启动器{
启动器名称不能超过16个字符，只能包括英文字符、数字以及下划线（联机）
没模组就看一眼版本隔离（模组）
游戏内存不能太小（小心崩溃）
报错后第一时间在启动器里的自动安装界面里把 OptiFine 去掉 https://gitcode.net/chearlai/kookfiles/-/raw/master/attachments/2023-05/19/3fLwoleY2Q0mq0e4.png （除了1.16.5以下）
> [!CAUTION]
> 重点看高级设置
> 如果没专业的人叫你动就别动
> 默认全是空的或关闭
> 本地库路径默认为预设
> 渲染器默认为“OpenGL”

# 不允许以下情况
![不许模仿！！！](https://f.cdn-static.cn/81470_17234455117292.png)

解决不了请加入此群666546887（手机还是别了，网易版绝对不可以）
此群仅处理 Minecraft 游戏报错崩溃问题和 HMCL 启动器相关疑难杂症，不处理挂端问题。（不处理PCL**启动器的问题**，不处理外挂）

> [!IMPORTANT]
> 补充
> 最近i18服务器不稳定，可能导致MC加载速度慢
> 近期正版服务器不稳定，可能无法登入
> 记得看crash开头的文件（里面有好东西）
> 
> # 每日任务做了吗？

**Comment by su2569** [comment]
排版显示出问题了wc

**Comment by su2569** [comment]
好了

**Comment by gaojiaobei** [comment]
给你点赞👍

**Comment by greenhoie** [comment]
什么？这里还能看到mc玩家
![屏幕截图 2024-08-12 192655](https://github.com/user-attachments/assets/b299e8c8-4faf-46ec-b4e8-2f7cbb6d2af3)

**Comment by yaotian-yyds** [comment]
这是绳网吧，那来的方块人٩(๛ ˘ ³˘)۶❤

**Comment by su2569** [comment]
其实是发上来当备忘录的[doge]

**Comment by su2569** [comment]
我梯子炸了（悲
